typescript/src/util.ts

```diff
@@ -258,3 +227,43 @@ export function processPathTemplate(
 
   return {fieldSend, messageRegex};
 }
+
+// Quick'n'dirty implementation of mkdirp to avoid pulling a dependency
+export async function mkdirp(directory: string) {
```

Comment:
Node 14 should have mkdirp built in, just set `recursive = true` for any `mkdir` operations.

Reply:
Oh nice! I target Node 18 so I'll make this change now, thank you!